### PR TITLE
fix(DB/Creature): Add sniffed waypoints to Fenissa the Assassin

### PR DESCRIPTION
**Pull request by Gultask**
Authored-By: neifion-00000000 <91289495+neifion-00000000@users.noreply.github.com>

## Changes Proposed:
-  Adds a third spawn
-  Adds waypoints to all spawns

## Issues Addressed:

- Closes 

## SOURCE:

TBC Classic

## Tests Performed:

- Tested ingame
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1676604656261986500.sql [hunk 1]
```diff
@@ -0,0 +1,117 @@
+--
+DELETE from `creature` where `guid`=77831 AND `id1`=22060;
+INSERT INTO `creature` (`guid`, `id1`, `id2`, `id3`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(77831, 22060, 0, 0, 530, 0, 0, 1, 1, 1, -2024.52, -10786.1, 81.9137, 0.664982, 3600, 0, 0, 751, 0, 2, 0, 0, 0, '', 0);
+
+UPDATE `creature_template_addon` SET `bytes1` = 0 WHERE (`entry` = 22060);
+
+-- Pathing for Entry: 22060
+SET @NPC := 77831;
+SET @path := @NPC * 10;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1676604656261986500.sql:10]: dont use path as it is a keyword in mysql. can cause issues. Use cpath or something

File: data/sql/updates/pending_db_world/rev_1676604656261986500.sql [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+UPDATE creature SET wander_distance=0,MovementType=2,position_x=-1970.9095,position_y=-10545.881,position_z=176.9451 WHERE guid=@NPC;
+DELETE FROM creature_addon WHERE guid=@NPC;
+INSERT INTO creature_addon (guid,path_id,mount,bytes1,bytes2,emote,visibilityDistanceType,auras) VALUES (@NPC,@path,0,0,1,0,0, '34189');
+DELETE FROM waypoint_data WHERE id=@path;
+INSERT INTO waypoint_data (id,point,position_x,position_y,position_z,orientation,delay,move_type,action,action_chance,wpguid) VALUES
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1676604656261986500.sql:15]: backticks

File: data/sql/updates/pending_db_world/rev_1676604656261986500.sql [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+-- 0x204CB04240158B0000003E0000187A4E .go xyz -1970.9095 -10545.881 176.9451
+
+-- Pathing for Entry: 22060
+SET @NPC := 77280;
+SET @path := @NPC * 10;
+UPDATE creature SET wander_distance=0,MovementType=2,position_x=-2062.9329,position_y=-10801.382,position_z=69.16798 WHERE guid=@NPC;
+DELETE FROM creature_addon WHERE guid=@NPC;
+INSERT INTO creature_addon (guid,path_id,mount,bytes1,bytes2,emote,visibilityDistanceType,auras) VALUES (@NPC,@path,0,0,1,0,0, '34189');
+DELETE FROM waypoint_data WHERE id=@path;
+INSERT INTO waypoint_data (id,point,position_x,position_y,position_z,orientation,delay,move_type,action,action_chance,wpguid) VALUES
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1676604656261986500.sql:63]: same

File: data/sql/updates/pending_db_world/rev_1676604656261986500.sql [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+-- 0x204CB04240158B0000003E00001879BC .go xyz -2062.9329 -10801.382 69.16798
+
+-- Pathing for Entry: 22060  Cryo Core
+SET @NPC := 77281;
+SET @path := @NPC * 10;
+UPDATE creature SET wander_distance=0,MovementType=2,position_x=-2070.419,position_y=-11257.795,position_z=68.73054 WHERE guid=@NPC;
+DELETE FROM creature_addon WHERE guid=@NPC;
+INSERT INTO creature_addon (guid,path_id,mount,bytes1,bytes2,emote,visibilityDistanceType,auras) VALUES (@NPC,@path,0,0,1,0,0, '34189');
+DELETE FROM waypoint_data WHERE id=@path;
+INSERT INTO waypoint_data (id,point,position_x,position_y,position_z,orientation,delay,move_type,action,action_chance,wpguid) VALUES
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1676604656261986500.sql:100]: same

File: data/sql/updates/pending_db_world/rev_1676604656261986500.sql (hunk 2 above)
Comment (Gultask) [reply data/sql/updates/pending_db_world/rev_1676604656261986500.sql]: Discord ate the backticks. I don't know how I didn't notice it

**Review by Nyeriah — APPROVED** [review]

**Pull request merged**